### PR TITLE
Add concerns about the currently listed mailing list and GitHub access.

### PR DESCRIPTION
**Pull request by NicholasRostant**
- Lists
- All
- The 
- Way
- Down

### CODE REVIEW

**Comment by NicholasRostant** [comment]
@CloudCoppola

File: committee/2017-03-03/agenda.md [hunk 1]
```diff
@@ -92,5 +92,16 @@ Claudio M Coppola (clc20)
   - AberDroneRacing 
   - AberRobotWars
   
+- Fate of the mailing list
+  - [This](https://groups.google.com/forum/#!forum/abercompsoc) is still listed on the website as the official mailing list.
+  - Not used for the year so far.
```
Comment (kragniz) [inline committee/2017-03-03/agenda.md]: Yeah, this should have been removed at the start of the year.

File: committee/2017-03-03/agenda.md [hunk 2]
```diff
@@ -92,5 +92,16 @@ Claudio M Coppola (clc20)
...
+  
+- Review Member level access to the GitHub
+  - If projects within the society are intended to live in the society GitHub members should probably have write access and not have to go through comittee members to merge a pull request.
+  - Certain repositiories should probably only have write access for committee members (e.g. official docs)
+    - Could employ GitHub's Teams feature to provide correct access levels.
+  - Members should probably be able to assign and label issues.
```
Comment (kragniz) [inline committee/2017-03-03/agenda.md]: Sadly not possible without giving members full write access, which sucks for any big project on github.

Comment (NicholasRostant) [inline committee/2017-03-03/agenda.md]: In which case is it possible leverage Teams to give full write on repos for projects/meetings for members then have the Management related repos locked down?

**Pull request merged**